python/doc/examples/data_analysis/smoothing_mixture.ipynb

```diff
@@ -0,0 +1,524 @@
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "fit.drawPDF()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see that there are two modes in the distributions. However, these two modes are somewhat unclearly distinguished. The first suggestion would be to increase the sample size. However, it might be interesting to see if the bandwidth selection rule can be better chosen: this is the purpose of the next section."
```

Comment:
```suggestion
    "We see that the distribution of the merged sample has two modes. However, these modes are not clearly distinct. To distinguish them, we could increase the sample size. However, it might be interesting to see if the bandwidth selection rule can be better chosen: this is the purpose of the next section."
```
I think keeping it simple is better.